src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/query.svelte

```diff
@@ -34,7 +37,15 @@
     return queryTypes;
   });
 
-  $: queryResult = queryType && getQuery({ namespace, workflow, queryType });
+  let queryResult: string;
+
+  const query = async (queryType: string) => {
```

Comment:
I feel a little iffy about an asynchronous statement that has side effects.

Reply:
changed to svelte based await